plugins/redis/lib_redis.ecllib

```diff
@@ -17,57 +17,77 @@
 
 
 EXPORT sync := SERVICE : plugin('redis'), namespace('RedisPlugin')
```

Comment:
Is it still sensible to call this sync (given there is now no other supported interface to redis)? If sync is something that would be meaningful to a redis user, then I don't mind keeping it...